templates/repo/issue/view_title.tmpl

```diff
@@ -123,6 +123,10 @@
 					·
 					{{ctx.Locale.TrN .Issue.NumComments "repo.issues.num_comments_1" "repo.issues.num_comments" .Issue.NumComments}}
 				</span>
+				{{if .MaybeFixed}}
+					{{$fixedStr := printf `<a href="%s">#%d</a>` .MaybeFixed.Issue.Link .MaybeFixed.Index}}
+					· <span>{{ctx.Locale.Tr "repo.issues.maybefixed" ($fixedStr|SafeHTML)}}</span>
```

Comment:
There should be no new SafeHTML in code anymore.

Reply:
0e0527456edf836b1e77b71f4ac1f3412b6ab08a